cmd/controller/main.go

```diff
@@ -133,8 +133,8 @@ func main() {
 		logger.Fatal("Could not create controller from environment or flags")
 	}
 
-	// if the kubeconfig fails BuildConfigFromFlags will try in cluster config
-	clientConf, err := clientcmd.BuildConfigFromFlags("", ctlConf.KubeConfig)
+	// if the kubeconfig fails InClusterBuildConfig will try in cluster config
+	clientConf, err := runtime.InClusterBuildConfig(logger.WithFields(logrus.Fields{}), ctlConf.KubeConfig)
```

Comment:
```suggestion
	clientConf, err := runtime.InClusterBuildConfig(logger, ctlConf.KubeConfig)
```

Should be sufficient, and same for the other instances where there is already a global logger in place.